pages/invitation/InvitationHandler.php

```diff
@@ -117,256 +110,29 @@ public static function getActionUrl(InvitationAction $action, Invitation $invita
     }
 
     /**
-     * Create an invitation to accept new role
-     *
+     * Create an invitation for a user to accept new roles
      * @throws \Exception
      */
-    public function invite(array $args, Request $request): void
+    public function userInvite(array $args, Request $request): void
```

Comment:
This function constitutes a security breach here. This handler does not have an `authorize` function. 

`accept` and `decline` functions depend on a provided key and id. The key is secret and is send only to the indented user. That is why this handler does not have any `authorization` code.

But having this  `userInvite` function, anyone can access any invitation I think.

If we can't solely depend on the API for the creation of the UI, I think we should create another handler here, with the appropriate `authorization` checks and policies.